### DISCUSSION TITLE
Not usable

### DISCUSSION THREAD

**Discussion by ad-fabiane** [bennypowers/nvim-regexplainer#29] [Q&A]
Hello Guys,

i wanted to try this amazing plugin.

But calling any of the provided Regexplain functions as well as the autocmd don't work.
I tried creating files with supported filetypes and several keybindings formats.

Is this error just in the latest release?

Best regards

**Comment by bennypowers** [comment]
Open vim, and type this into the command line

```vim
:redir @">|silent TSInstallInfo|redir END|enew|put
```
Then press enter, then save the file and paste its contents here

**Comment by ad-fabiane** [comment]
I installed:
- c                 [✓] installed
- http              [✓] installed
- json              [✓] installed
- regex             [✓] installed
with Treesitter.
Everything else is not installed.

**Comment by bennypowers** [comment]
send me a file for which regexplainer does not work as expected

**Comment by ad-fabiane** [comment]
_example.ts_
`/hello @bennypowers/;`

**Comment by bennypowers** [comment]
you have to install the typescript treesitter grammar

**Comment by ad-fabiane** [comment]
ah that was the problem.
Thank you for your quick responses and your help <3

**Comment by bennypowers** [comment]
Glad to help, hope you like the plugin

**Comment by ad-fabiane** [comment]
i will
